third_party/blink/renderer/core/workers/dedicated_worker_global_scope.cc

```diff
@@ -369,7 +369,6 @@ void DedicatedWorkerGlobalScope::postMessage(ScriptState* script_state,
                                              const PostMessageOptions* options,
                                              ExceptionState& exception_state) {
   Transferables transferables;
-  recordreplay::AutoAssertBufferAllocations autoAsserts("TT-492-1286");
```

Comment:
shouldn't these autoAssert removals be part of a TT-1415 PR?

Reply:
No. These should still trigger if there are codepaths that diverge on serialization.
* The Assert I removed in TT-1415 is *in front of* your change. They trigger even if your patch worked.
* These Asserts are *behind* your change, so they would not trigger for all the instances where your patch worked.
* What I'm doing here is moving the same Asserts into a more central location because, as you can see from the linked comment, we have mismatches coming from other places as well, and I just want to make sure we got all of those covered.

Reply:
hm, I guess maybe it would have been more appropriate in my eye to combine the two PRs under the same issue in that case.  The title of the other (v8) PR was "disable 492 asserts", after all.  It's much harder (for me at least) to follow the flow when it's split up like this.

Reply:
Its entirely different changes. We can discuss on Monday if you don't feel that that is the case.
But yeah the title could have been chosen better.